.github/workflows/build.yml

```diff
@@ -64,15 +64,13 @@ jobs:
           export GOARCH=${{ matrix.arch }}
           export CGO_ENABLED=1
           CC=${{ matrix.cc }} go build -o $TARGET_NAME${{ matrix.ext }} cmd/massa-wallet/thyra-plugin-wallet.go
+      - name: Zip binary
+        run: zip $TARGET_NAME.zip $TARGET_NAME${{ matrix.ext }}
       - name: Upload artifacts
-        if: ${{ inputs.tag_name == '' && github.ref_name == 'main' }}
         uses: actions/upload-artifact@v3
         with:
-          name: thyra-plugin-wallet_${{ matrix.target }}-${{ matrix.arch }}
-          path: ${{ env.TARGET_NAME }}${{ matrix.ext }}
-      - name: Zip binary
-        if: ${{ inputs.tag_name != '' }}
-        run: zip $TARGET_NAME.zip $TARGET_NAME${{ matrix.ext }}
+          name: ${{ env.TARGET_NAME }}.zip
+          path: ${{ env.TARGET_NAME }}.zip
```

Comment:
There is a probleme here. upload artifact action takes some files and put it into a zip.
So here you are creating a zip into a zip

Reply:
I removed the commit that bring this change so that we can merge it asap